cellxgene_schema_cli/tests/test_validate.py

```diff
@@ -730,33 +777,51 @@ def test__validate_images_image_is_shape_error(self, image_name):
             "for example) or 4 (RGBA color model for example) for its last dimension" in validator.errors[0]
         )
 
-    def test__validate_images_hires_max_dimension_greater_than_error(self):
+    @pytest.mark.parametrize(
+        "assay_ontology_term_id, hi_res_size, image_max",
+        [
+            ("EFO:0022858", 2001, SPATIAL_HIRES_IMAGE_MAX_DIMENSION_SIZE),
+            ("EFO:0022860", 4001, SPATIAL_HIRES_IMAGE_MAX_DIEMSNION_SIZE_VISIUM_11MM),
```

Comment:
suggestion: this check is that the largest dimension in the hires array IS SPATIAL_HIRES_IMAGE_MAX_DIMENSION_SIZE or SPATIAL_HIRES_IMAGE_MAX_DIEMSNION_SIZE_VISIUM_11MM depending on the assay (rather than a check that the largest dimension is <= SPATIAL_HIRES_IMAGE_MAX_DIMENSION_SIZE)

so, it might be good to have a test making sure those checks don't get flipped--i.e. "EFO:0022858" (not Visium 11MM) FAILS with largest dimension SPATIAL_HIRES_IMAGE_MAX_DIEMSNION_SIZE_VISIUM_11MM and vice-versa with the Visium 11MM check.

Reply:
done: added equality check and check that the two don't get flipped.